common-lib/src/main/scala/com/gu/mediaservice/model/ImageMetadata.scala

```diff
@@ -7,17 +7,17 @@ import play.api.libs.json._
 
 case class ImageMetadata(
   dateTaken:           Option[DateTime] = None,
-  description:         Option[String]   = None,
-  credit:              Option[String]   = None,
+  description:         Option[String]   = None, //
```

Comment:
what are the empty comments about?

Reply:
will remove them, i was ensuring myself which fields are editable

Reply:
done